src/Components/Server/test/Circuits/RemoteJSDataStreamTest.cs

```diff
@@ -154,10 +155,104 @@ public async void ReceiveData_ProvidedWithOutOfOrderChunk_SimulatesSignalRDiscon
             Assert.Equal("Out of sequence chunk received, expected 5, but received 7.", ex.Message);
         }
 
+        [Fact]
+        public async void ReceiveData_NoDataProvidedBeforeTimeout_StreamDisposed()
```

Comment:
don't use async void (it's almost never correct to do so)

Reply:
```suggestion
        public async Task ReceiveData_NoDataProvidedBeforeTimeout_StreamDisposed()
```